platform/mbed_lib.json

```diff
@@ -64,6 +64,11 @@
         "max-error-filename-len": {
             "help": "Sets the maximum length of buffer used for capturing the filename in error context. This needs error-filename-capture-enabled feature.",
             "value": 16
+        },
+        
+        "error-decode-http-url-str": {
+            "help": "HTTP URL string for ARM Mbed-OS Error Decode microsite",
+            "value": "\"\\nFor more info, goto: https://armmbed.github.io/mbedos-error/?error=0x%08X\""
```

Comment:
I would suggest either adding a space in `goto` or replacing it with `visit`.

Reply:
"visit" sounds better, let me make that change.